ocaml/xenopsd/lib/cancellable_subprocess.ml

```diff
@@ -76,16 +76,17 @@ let run (task : Xenops_task.task_handle) ?env ?stdin fds
             raise (Spawn_internal_error (err, out, Unix.WEXITED n))
         | Unix.WSTOPPED n ->
             raise (Spawn_internal_error (err, out, Unix.WSTOPPED n))
-        | Unix.WSIGNALED n ->
+        | Unix.WSIGNALED s ->
```

Comment:
Why renaming `n` to `s`? In the other cases it's `n`.

Reply:
s for signal, n for (exit) number